src/py42/clients/archive.py

```diff
@@ -144,6 +145,8 @@ def stream_to_device(
                 key archive security. Defaults to None.
             show_deleted (bool, optional): Set to True to include deleted files when restoring a directory.
                 Defaults to None.
+            overwrite_existing_files (bool, optional): Set to True when using ORIGINAL_LOCATION for the
+                restore_path argument to overwrite any existing files part of the restore.
```

Comment:
to overwrite any existing files with the restored data. If False (the default), any existing files that match a path being restored will first get renamed.